src/basic/env-util.c

```diff
@@ -183,36 +183,34 @@ static int env_append(char **r, char ***k, char **a) {
         return 0;
 }
 
-char **strv_env_merge(size_t n_lists, ...) {
+char** _strv_env_merge(char **first, ...) {
         _cleanup_strv_free_ char **ret = NULL;
```

Comment:
maybe rename this "merged" or so, away from "ret", as we usually use that for other purposes